tn_msdp_api_v1.yml

```diff
@@ -0,0 +1,1736 @@
+        and pre-production endpoints - these correspond to the GDS’s production and pre-production services, and can be used as desired.<br  />
+        <h2>Trip Ninja API URLs</h2>
+        
+        Pre-Production: https://preprodapi.tripninja.io
+        
+        For security, Trip Ninja’s servers are only accessible from whitelisted IPs. Please contact your account manager to whitelist IPs you
+        will be using.
+        
+        Before you start ensure that:<br  />
+            - IP addresses have been provided to be whitelisted for our servers.<br  />
+            - PCC / OfficeID emulation via Trip Ninja’s PCC/OfficeID has been set up.<br  />
+            - API username and password have been provided by Trip Ninja.<br  />
+        
+        <h2>Authentication</h2>
+        Trip Ninja uses Basic Authentication standards. Simply encode your username and password string using base64 and pass it in the
+        authorization headers. See the python example below on how this is done. In the example, the payload and API endpoint url are not
+        shown.
+        
+        Python Example:
+        
+        <pre>
+        import base64
+        auth = base64.b64encode("USERNAME:PASSWORD")
+        headers = {
+            'authorization': "Basic "+ auth,
+        }
+        response = requests.post(url, headers=headers, data=json.dumps(payload))
+        </pre>
+
+    x-logo:
+        url: 'https://s3.amazonaws.com/tn-api-docs/trip_ninja_logo.png'
+        altText: Trip Ninja logo
+        href: 'https://www.tripninja.io/'
+
+servers:
+    -   url: https://preprodapi.tripninja.io
+        description: Pre-Production server
+
+paths:
+    /msdp/search/{endpoint}:
+        post:
+            parameters:
+                -   in: path
+                    name: endpoint
+                    required: true
+                    schema:
+                        type: string
+                        enum: [ preprod, prod ]
+                    description: Parameter toggles data source production and pre-production endpoints
+            summary: "MSDP Search"
+            description: |
+                This endpoint initiates a msdp search for one or more flights and one or more hotels, and returns a list of available flights and hotels options. A separate endpoint needs to be called to obtain information about flight details, hotel room types, etc.
+            operationId: Search
+            requestBody:
+                content:
+                    application/json:
+                        schema:
+                            $ref: "#/components/schemas/Requests/MSDPSearchRequest"
+            responses:
+                200:
+                    description: Success
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/Responses/MSDPSearchResponse"
```

Comment:
Might be useful to return OTA and QT format of response side by side instead of altogether. I am looking todo the same thing in https://app.clickup.com/t/14197839/ENG-672. 

Here is a source on how todo that : https://stackoverflow.com/a/48427558
Look at the OpenAPI 3.x section.

This is what it looks like to have more then 1 response on my branch. 

![Screenshot 2022-12-05 at 4 07 38 PM](https://user-images.githubusercontent.com/47077094/205752774-22d1e331-ab4a-40f3-8193-c9c822890482.png)

Reply:
this is pretty slick !

Reply:
ooh boy, thats very advanced and it looks amazing, will take a look when the the higher priority tickets are finished

Reply:
@Sanpele Updated!